File: src/Backend/MovieBackend/Services/MovieCatalog/MovieCatalog.Domain/Entities/Movie/Movie.cs
namespace MovieCatalog.Domain.Entities.Movie
{
	public class Movie
	{
		public int ID { get; set; }
		public string? Title { get; set; }
		public string MovieImage { get; set; }
		public string? Description { get; set; }
		public DateRelease? ReleaseDate { get; set; }

	}
}

File: src/Frontend/MovieLibrary/src/app/models/movie.ts
export class Movie {
  id!: number;
  title!: string;
  description!: string;
  movieImage!: string;
  averageRating!: string;
  releaseDate!: ReleaseDate;
}

export class ReleaseDate {
  day!: number;
  month!: number;
  year!: number;
}
